### DISCUSSION TITLE
Patreon account

### DISCUSSION THREAD

**Discussion by nagaitsev** [osy86/HaC-Mini#559] [General]
@osy86 Do you have patreon account? You do great job and I want to thank you that way. I understand that it may be just a cup of coffee, but it's better than nothing :)

**Comment by msedekhyh** [comment]
would love to do the same.. nothing like those morons in tony mac

**Comment by lmshsqlc** [comment]
Would love to do

**Comment by jposemescouilles** [comment]
I would also like to give something, your work is great!

**Comment by TayshunAlly** [comment]
I too would donate you a beer!

**Comment by hongkongkiwi** [comment]
Me too

**Comment by chirimar** [comment]
+100

**Comment by lmshsqlc** [comment]
So anyway this issue hasn’t been solved and apparently osy86 is well aware of this problem.

**Comment by chirimar** [comment]
New release. New Catalina update. 
Everything still works seamlessly. 
Everything feels better. 

@osy86 needs to start taking thanks/gifts! Kudos!

**Comment by mikeanthonywild** [comment]
@osy86, since GitHub obfuscates commit author email addresses, I just wanted to take the time to write and thank you (and the other contributors) for your hard work. I picked up a Hades Canyon NUC on a whim to play some games, fully expecting to have to keep my ageing 2012 Mac mini for work. Much to my surprise, macOS is now running on it without a hitch. In fact, 
the hardest part of the whole process was getting a Windows installer USB created — not something that I expected!

I particularly appreciate that in addition to the very concise installation guide, you have also written in detail about the work that was actually done to get macOS running on these devices.

**Comment by cptn-cosmo** [comment]
I have to agree! Discovered this project yesterday and am blown away with how easy and seamless everything is! Please provide a way for us to give something back for your hard work! @osy
